src/components/client/data/Data.tsx

```diff
@@ -61,18 +69,55 @@ export const Data: React.FC<{ data: IInstructionData }> = ({
           });
         }}
       >
-        <TabList>
-          <Tab {...tabStyle} rounded="md">
-            Borsh{" "}
-            {isAnchor && <LockIcon ml="1" w="2.5" color={lockedIconColor} />}
-          </Tab>
-          <Tab {...tabStyle} rounded="md" isDisabled={isAnchor}>
-            Buffer Layout
-          </Tab>
-          <Tab {...tabStyle} rounded="md" isDisabled={isAnchor}>
-            Raw
-          </Tab>
-        </TabList>
+        <Box display="flex">
+          <TabList>
+            <Tab {...tabStyle} rounded="md">
+              Borsh{" "}
+              {isAnchor && <LockIcon ml="1" w="2.5" color={lockedIconColor} />}
+            </Tab>
+            <Tab {...tabStyle} rounded="md" isDisabled={isAnchor}>
+              Buffer Layout
+            </Tab>
+            <Tab {...tabStyle} rounded="md" isDisabled={isAnchor}>
+              Raw
+            </Tab>
+          </TabList>
+          {
+            format === "raw" ? <Box ml="auto">
```

Comment:
This is a leaky abstraction. This whole logic should be in the `RawData` component since it's specific to do that.

Don't stress if it means the button won't be with in the same line as tabs. We rather have good abstractions :)

Reply:
* the button is contained within RawData